src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs

```diff
@@ -485,7 +486,8 @@ public async void UpdateReviewBackground()
             {
                 try
                 {
-                   await UpdateReviewAsync(review);
+                    await UpdateReviewAsync(review);
+                    Thread.Sleep(5000);
```

Comment:
use Task.Delay

Reply:
Let's also wrap every update into Appinsights scope:  https://docs.microsoft.com/en-us/azure/azure-monitor/app/custom-operations-tracking#process

Reply:
So we can see them in logs later

Reply:
Thanks. Updated PR with these changes.